src/inputType/WheelInput.ts

```diff
@@ -1,17 +1,17 @@
-import { InputObserver } from "./../InputObserver";
 import { $ } from "../utils";
 import { toAxis, IInputType, IInputTypeObserver } from "./InputType";
-import { Axis } from "../AxisManager";
 
 export interface WheelInputOption {
 	scale?: number;
+	releaseThreshold?: number;
```

Comment:
As the term `threshold` is used like input threshold to some px in other inputs, it sounds more like input threshold to me,
How about `releaseDelay` or something?

Reply:
The team `releaseDelay` looks better than `releaseThreshold`, since it uses millisecond unit.